flow/designs/asap7/eth_top/config.mk

```diff
@@ -0,0 +1,50 @@
+$(info [INFO-FLOW] Ethernet Design)
+
+DESIGN_DIR                   := $(realpath $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST)))))
+DESIGN_PDK_HOME              := $(realpath $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST)))))
+
+export DESIGN_NAME            = eth_top
+export DESIGN_NICKNAME        = eth_top
+export DESIGN                 = eth_top
```

Comment:
Why create another variable?

Reply:
@maliberty  DESIGN_NAME used inside scripts/*.tcl for reference. DESIGN_NICKNAME or DESIGN variable used to point src directory path.

Reply:
`DESIGN` is not used anywhere in the flow. And `DESIGN_NICKNAME` is set to `DESIGN_NAME` if it doesn't exist. Therefore, only setting `DESIGN_NAME` is required here.